src/battle_script_commands.c

```diff
@@ -7271,19 +7589,20 @@ static void atkA1_counterdamagecalculator(void)
     }
 }
 
-// a copy of atkA1 with the physical -> special field changes
-static void atkA2_mirrorcoatdamagecalculator(void)
+static void Cmd_mirrorcoatdamagecalculator(void) // a copy of Cmd with the physical -> special field changes
```

Comment:
Find and replace mistake here with `atkA1`. Also in general these comments should go above the function (I know this was synced from pokeemerald)